src/libs/Navigation/AppNavigator/WorkspaceSettingsModalStackNavigator.tsx

```diff
@@ -0,0 +1,83 @@
+import {createStackNavigator} from '@react-navigation/stack';
+import React, {useMemo} from 'react';
+import useThemeStyles from '@hooks/useThemeStyles';
+import SCREENS from '@src/SCREENS';
+
+const StackNavigator = createStackNavigator();
+
+function WorkspaceSettingsModalStackNavigator() {
+    const styles = useThemeStyles();
+    const screenOptions = useMemo(() => ({cardStyle: styles.navigationScreenCardStyle, headerShown: false}), [styles]);
```

Comment:
You're missing the `cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS` that's present in the `defaultSubRouteOptions` in `createModalStackNavigator`

Reply:
fixed 🙇